focus-areas/living-wage/Job-Opportunities.md

```diff
@@ -0,0 +1,34 @@
+# Job Opportunities
```

Comment:
I thought we already merged #50 and [this file](https://github.com/chaoss/wg-value/blob/master/focus-areas/living-wage/Job-Opportunities.md) already exists...

Can you please rebase your branch to only submit the new changes?